app.js

```diff
@@ -29,30 +38,26 @@ navigator.geolocation.getCurrentPosition(getLocation, handleError);
 // Response format is Json by default.
 // https://openweathermap.org/current --free
 
-const KEY = "3137461397c70af0e13c77fda97afa11"
+const KEY = '3137461397c70af0e13c77fda97afa11';
 
-function getTemp(latitude, longitude){
-    let api = `http://api.openweathermap.org/data/2.5/weather?lat=${latitude}&lon=${longitude}&appid=${KEY}&units=metric`
+function getTemperature(latitude, longitude) {
+    const api = `https://api.openweathermap.org/data/2.5/weather?lat=${latitude}&lon=${longitude}&appid=${KEY}&units=metric`;
     //the fetch() method instructs the web browsers to send a request to a URL.
     //less than60 calls/min
     fetch(api)
-    .then(response => {
-        // handles response
-        // console.log(response.statusText); when response.status is 200, it is OK.
-        if (response.status != 200){
-            console.log("there is a problem with the request")
-        }else{
-            // data --> Promise object represents the eventual completion (or failure) 
+        .then((response) => {
+            // data --> Promise object represents the eventual completion (or failure)
             // of an asynchronous operation and its resulting value.
-            let data = response.json().then(data=> {
-            console.log(`Request succeed - json: ${data}`)    
-            document.getElementById("temperature").innerHTML = `The 
-            temperature in your location is ${data.main.temp.toFixed(2)}C°`}) 
-        }        
-    })
-    .catch(error => {
-        // handles errors
-        console.warn(`error: ${error.message}, weather API`)  
-    });
+            // console.log(response);
+            return response.json();
```

Comment:
You don't need to change it right now, but here is an explanation about arrow functions https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions
Which you're using already anyway :) so after you read it you an take a look again on this part of a code and find a place where you can simplify it!

Reply:
changed 👍
I'm guessing it is this part.
.then (response => response.json())